Added a split example (#884)
apache/pulsar-site: docs/concepts-broker-load-balancing-concepts.md

Step 3: split bundles at the specific boundaries from step 2.

For how to split bundles manually, please refer to [split-bundle](https://pulsar.apache.org/reference/#/3.2.x/pulsar-admin/namespaces?id=split-bundle) admin command.

> 

Reviewer: This link would point to a hardcoded versioned URL, where we should use the URL relative to each document version.

Author: thanks. updated

Example:

`pulsar-admin namespaces split-bundle -b LARGEST -san topic_count_equally_divide -u tenant/largest_namespace`

> * Specify a specific bundle at -b, if you already know the target bundle to split.
> 

Reviewer: ```suggestion
If you already know the target bundle to split, you can specify it using the `-b` flag:

`pulsar-admin namespaces split-bundle --bundle 0x00000000_0xffffffff test-tenant/namespace1`
```
